renovate.json

```diff
@@ -15,18 +15,6 @@
         "/^Microsoft\\.Bcl\\.AsyncInterfaces$/"
       ]
     },
-    {
```

Comment:
Why do you want to update the dependency?
It should be set to the minimum so the analyzer can be used in more context (older VS, Rider, dotnet build)

Reply:
Hmm, we don't want to update? HQ is restoring the previous behavior which already was to set the minimal requirements for dependencies.

Reply:
Sorry, I didn't read the diff correctly...

The previous code was more specific. The project that is released doesn't update the dependencies to be at the lowest version possible, but the tests could be updated to test with the latest version. Do we want to update this logic?